config/pipeline.yaml

```diff
@@ -224,6 +231,9 @@ jobs:
       compiler: gcc-10
       cross_compile: 'arm-linux-gnueabihf-'
       defconfig: multi_v7_defconfig
+    rules:
+      tree:
+        - '!android'
```

Comment:
Just double checking: do you really intend to build arm32 kernels from the mediatek tree?

Reply:
Yep, there are arm32 mediatek boards, and they still get patches landed through that tree.

Reply:
OK, thanks for confirming!